### PR TITLE
Release notes 19.2.0

### PR DESCRIPTION
**Pull request by kritika-singh3**

### CODE REVIEW

File: source/partials/release_notes/_release-19-2-0.md.erb [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+* The notification plugin extension version 3 has been deprecated. This version will be removed in a release scheduled for Mar 2019. Plugin developers should use version 4 of the notification plugin api.
+* The analytics plugin extension version 1 has been deprecated. This version will be removed in a release scheduled for Mar 2019. Plugin developers should use version 2 of the analytics plugin api.
+* The Get Server Info processor version 1 has been deprecated. This version will be removed in a release scheduled for April 2019. Plugin developers should use version 2 of the Get Server Info processor.
+* The Dashboard API v1 has been deprecated. This version of the API will be removed in a release scheduled for March 2019. Version 2 of the API is available, and users are encouraged to use it.
```
Comment (GaneshSPatil) [inline source/partials/release_notes/_release-19-2-0.md.erb]: dashboard API v2?

Comment (kritika-singh3) [reply source/partials/release_notes/_release-19-2-0.md.erb]: Fixed.

File: source/partials/release_notes/_release-19-2-0.md.erb [hunk 2]
```diff
@@ -0,0 +1,131 @@
+<h4> Personal Access Tokens </h4>
+
+Personal access tokens can be utilized by the GoCD users to access GoCD API(s) without passing in their credentials(username & password). This allows user logged in GoCD server via web based plugins to generate a token and use that to make an API call. For this to happen, the plugin must have support for authorization extension v2.
```
[the remark this answers is not in the document]
Comment (kritika-singh3) [reply source/partials/release_notes/_release-19-2-0.md.erb]: Don't we need to specify for plugin users that the plugin needs to be updated to support authorization extension v2?

File: source/partials/release_notes/_release-19-2-0.md.erb [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+<h4> Improved pipeline pause information </h4>
+Now user will see time when a pipeline was pased along with pause reason.
+
+<figure class='shrink-figure-100'>
+    <%= image_tag 'releases/19.2.0/pipeline-pause.png', alt: 'Pipeline Info when Paused' %>
+</figure>
+<br/>
+
+<h4> Improved User Management SPA </h4>
```
[the remark this answers is not in the document]
Comment (kritika-singh3) [reply source/partials/release_notes/_release-19-2-0.md.erb]: Done

File: source/partials/release_notes/_release-19-2-0.md.erb [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+Now admins will be able to bulk delete users and to give/revoke system-admin priviliges to a user.
```
[the remark this answers is not in the document]
Comment (kritika-singh3) [reply source/partials/release_notes/_release-19-2-0.md.erb]: Done

File: source/partials/release_notes/_release-19-2-0.md.erb [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+<figure class='shrink-figure-100'>
+    <%= image_tag 'releases/19.2.0/user-management-spa.png', alt: 'User Management SPA' %>
+</figure>
+<br/>
+
+
+<h4> Config repo improvements </h4>
+
+* <%= link_to_commit 'b2c63848fd3abaa5c4812ea3bf95154e9a6d8426', 'Parse and merge ConfigRepos on any change in config repo configuration.', 5792 %>
+* <%= link_to_pull_request 5828, 'Allow user\'s to configure empty secure variables' %>
```
[the remark this answers is not in the document]
Comment (kritika-singh3) [reply source/partials/release_notes/_release-19-2-0.md.erb]: Using `link_to_commit`.

**Pull request merged**